index.js

```diff
@@ -4,7 +4,36 @@
  * Depending on the operation, either add up all of the numbers or subtract all of the numbers, from left to right.
  * @returns {number} The result of either adding all numbers or subtracting all numbers, depending on the arguments added to the command line.
  */
-function calculator() {}
+function calculator() {
+console.log(process.argv)
+    if(!process.argv[2]){
+        return `No operation provided...`
+    } else if(process.argv.length === 3){
+        return `No numbers provided...`
+    } 
+
+
+    if (process.argv[2] === `plus`){
```

Comment:
The code in this else if block and the one on line 24 are very similar with few differences. How can you refactor this code to be less repetitive??